client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/notices/notices.tsx

```diff
@@ -0,0 +1,38 @@
+import { SnackbarList, withNotices } from '@wordpress/components';
+import i18n from 'i18n-calypso';
+import { ReactNode } from 'react';
+import type { Pattern } from '../types';
+import './notices.scss';
+
+type NoticesProps = {
+	noticeList: withNotices.Props[ 'noticeList' ];
+	noticeOperations: withNotices.Props[ 'noticeOperations' ];
+	noticeUI?: ReactNode;
+};
+
+const Notices = ( { noticeList, noticeOperations }: NoticesProps ) => {
+	const onRemoveNotice = ( id: string ) => {
+		noticeOperations.removeNotice( id );
+	};
+
+	return <SnackbarList notices={ noticeList } onRemove={ onRemoveNotice } />;
+};
+
+export const getNoticeContent = ( action: string, pattern: Pattern ) => {
+	const actions: { [ key: string ]: any } = {
+		add: i18n.translate( 'Pattern "%(categoryName)s" added.', {
```

Comment:
Other ideas for the notice copy: 

- `Pattern from category "Images" added.`
- `Pattern from category "Images" added to the Homepage.`